node/core/runtime-api/src/cache.rs

```diff
@@ -323,6 +331,21 @@ impl RequestResultCache {
 		self.session_info.insert(key, ResidentSizeOf(value));
 	}
 
+	pub(crate) fn session_ee_params_by_parent_hash(
+		&mut self,
+		key: (Hash, Hash),
```

Comment:
nit: I'm confused why `key` is `(Hash, Hash)` here when it is `Hash` below in `cache_session_ee_params_by_parent_hash`.

Reply:
I get what your feeling 'cause I was so much confused by how runtime API cache is organized too :)
The first hash is always a relay parent hash which sets the context of a request, that is, an entry is cached for the time of current block only (as far as I understood it, I'm still not sure I got it 100% right). The second hash is a relay parent hash for which EE params are requested. That is, we're currently at block 1200 (1st hash) and we're requesting EE params for the session which was effective at the time when block 1180 was current (2nd hash).

I think this part will be reworked anyway, as @eskimor pointed out that we do not need a separate mapping from relay parent hash to session index, as it can be requested with `session_index_for_child` call.